cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/FileLocalizer.java

```diff
@@ -64,17 +65,16 @@ public void run(String[] args) throws Exception {
 
     // Localize the runtime config jar
     URI uri = URI.create(args[0]);
-    Location runtimeConfigLocation;
-    if (context.getLocationFactory().getHomeLocation().toURI().getScheme().equals(uri.getScheme())) {
-      runtimeConfigLocation = context.getLocationFactory().create(uri);
-    } else {
-      runtimeConfigLocation = new LocalLocationFactory().create(new File(uri).toURI());
-    }
+    Location runtimeConfigLocation = new LocalLocationFactory().create(new File(uri.getPath()).toURI());
+
+    masterEnv.fetchFile(uri, new File(runtimeConfigLocation.toURI().getPath()));
 
-    Path runtimeConfigDir = expand(runtimeConfigLocation, Paths.get(Constants.Files.RUNTIME_CONFIG_JAR));
+    Path runtimeConfigDir = expand(runtimeConfigLocation,
+                                   Paths.get(org.apache.twill.internal.Constants.Files.RUNTIME_CONFIG_JAR));
 
-    try (Reader reader = Files.newBufferedReader(runtimeConfigDir.resolve(Constants.Files.TWILL_SPEC),
-                                                 StandardCharsets.UTF_8)) {
+    try (Reader reader = Files.newBufferedReader(
+      runtimeConfigDir.resolve(org.apache.twill.internal.Constants.Files.TWILL_SPEC),
```

Comment:
Why need to use long class name?

Reply:
Fixed. Used to conflict with constants from cdap-common package.